packages/genesys-spark-tokens/data/tokens.json

```diff
@@ -3508,6 +3508,40 @@
         "type": "spacing"
       }
     },
+    "z_index": {
+      "0" : {
+        "value": "0",
+        "type": "zindex"
+      },
+      "1" : {
+        "value": "1",
+        "type": "zindex"
+      },
+      "100" : {
+        "value": "100",
+        "type": "zindex"
+      },
+      "200" : {
+        "value": "200",
+        "type": "zindex"
+      },
+      "300" : {
+        "value": "300",
+        "type": "zindex"
+      },
+      "400": {
+        "value": "400",
+        "type": "zindex"
+      },
+      "500" : {
+        "value": "500",
+        "type": "zindex"
+      },
+      "600" : {
+        "value": "600",
+        "type": "zindex"
+      }
+    },
```

Comment:
@conor-darcy does the figma tokens plugin sync both ways or how can we add tokens from dev to UX?